prime-dotnet-webapi/Controllers/GisController.cs

```diff
@@ -0,0 +1,210 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+using Prime.Services;
+using Prime.Models.Api;
+using Prime.ViewModels.Parties;
+
+namespace Prime.Controllers
+{
+    [Produces("application/json")]
+    [Route("api/[controller]")]
+    [ApiController]
+    public class GisController : ControllerBase
```

Comment:
These endpoints are not authenticated; I assume that is on purpose until we have the MOH Keycloak integration working?